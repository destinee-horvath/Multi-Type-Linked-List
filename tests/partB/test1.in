NEW 4
5
times
6.0
30
INSERT 0 3 equals