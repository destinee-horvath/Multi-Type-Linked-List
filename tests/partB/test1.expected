List 0: 5 -> times -> 6.0 -> equals -> 30